### ISSUE TITLE
Changing the `controller` property function name?

### ISSUE THREAD

**Issue by danjford** [lodestarjs/lodestar-ractive#33]
At the moment if you're not separating out the controllers into separate files, the controller object will look like the following example, where I'm not sure I like the repetition of the attribute `controller`:

```
controller: {

  controller: function() { },

  actions: {},

  observe: {},

  observeOnce: {}
}

```

Maybe we could change the controller function to something like `onLoad` or `init`?

**Comment by danjford** [comment]
@xCausxn, @silverlight513 and anybody else, any suggestions?

**Comment by jamescauson** [comment]
I agree, `init` or `methods` something like that to make the flow cleaner

**Comment by danjford** [comment]
@xCausxn, I remember that Ember uses `setupController` which I guess does what is says on the tin... but I am leaning towards `init`.

**Comment by danjford** [comment]
@xCausxn do you prefer `onInit` or just simply `init`? The property will be executed when the controller is initialised after the Ractive View has been activated.

```
controller: {

  init: function() { },
  // OR
  onInit: function() {},

  actions: {},

  observe: {},

  observeOnce: {}

}
```

**Comment by xCausxn** [comment]
Hard choice but I would probably say onInit

**Comment by danjford** [comment]
Changes have been made on [my branch](https://github.com/danjford/lodestar-ractive/tree/development), I will merge it in once tests & coverage has been added.

**Comment by danjford** [comment]
Actually @xCausxn, maybe not `onInit` as I've noticed that Ractive has an `oninit` event so it could become confusing when using components etc... Maybe `onSetup`?

**Comment by xCausxn** [comment]
`run` or `onSetup`

**Comment by pantchox** [comment]
why not just call it 'main'

**Comment by silverlight513** [comment]
`onControllerLoad` might be another option.

**Comment by xCausxn** [comment]
Could change everything to ES6 classes, and then run babel over it.  Same way angular is doing it and I find it really nice, constructor can then be utilised, then anything else you need make a method for it.

**Comment by danjford** [comment]
There is already a [nice flow](https://github.com/lodestarjs/lodestar-ractive/blob/gh-pages/src/js/controllers/IndexController.js) in place using ES6 modules.

I don't know if it makes sense to switch everything to a flow revolving around ES6 classes as Lodestar-Ractive follows the MVC pattern whereas Angular 2 focuses much more on using components which contain their own logic rather than having controllers which are conceptually different.
